contracts/test/CompoundGovernor.t.sol

```diff
@@ -93,6 +102,21 @@ contract Propose is CompoundGovernorTest {
         );
         _submitProposal(_proposer, _proposal);
     }
+
+    function test_RevertIf_ProposerIsBelowThreshold(address _proposer) public {
```

Comment:
could also use a... `test_RevertIf_ExpiredWhitelistedAccountIsBelowThreshold` just to make sure we're covering the breadth of the feature

Reply:
added https://github.com/ScopeLift/compound-governance-upgrade/pull/89/commits/7a94bff9de6fa0d0bdf78d3960c3f94ca7efb9fd